test/fast/Unit tests/nvm_get_node_from_pkg_json

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+die () { echo "$@" ; exit 1; }
+
+\. ../../../nvm.sh
+
+# TODO add more test cases
+TEST_CASES=(
+'{\n\t"engines": {\n\t\t"node":"1.1.1"\n\t}\n}"' '1.1.1'
```

Comment:
On second thought, if we store each test case in its own example package.json file and make them human readable, it'd be difficult to see which ones are testing which type(s) of spacing. Also I can think of at least 25 test cases this should probably have, so I think it'd be easier to get the full picture of all the tests if they were listed as they currently are in a single file. What do you think?